packages/cubejs-client-core/index.d.ts

```diff
@@ -560,6 +560,7 @@ declare module '@cubejs-client/core' {
      * @returns An array of columns
      */
     tableColumns(pivotConfig?: PivotConfig): TableColumn[];
+    totalRow(pivotConfig?: PivotConfig):{ [key: string]: number };
```

Comment:
As `totalRow` returns the first item from the `chartPivot` array I believe the return type should be `ChartPivotRow`

Reply:
I got it 😄.  Let me update that right away

Reply:
Updated !!! Could you please take another look?